R/fwise.R

```diff
@@ -136,13 +133,23 @@ tf_crosscov <- function(x, y, arg = tf_arg(x)) {
   y_mean <- tf_integrate(y_)/length
   x_c <- x_ - x_mean
   y_c <- y_ - y_mean
-  tf_integrate(x_c * y_c)
```

Comment:
seems to me this line *should* have simply been 
`tf_integrate(x_c * y_c) / length` (see definition in roxygen block) ? 

IMO, after this change `tf_crosscor` would not need to be changed and no additional helper function needs to be introduced.